src/memefactory/server/graphql_resolvers.cljs

```diff
@@ -361,23 +361,8 @@
          param-changes-result (paged-query param-changes-query
                                            first
                                            (when after
-                                             (js/parseInt after)))
-         ret (if-not (= 0 (:total-count param-changes-result))
-               param-changes-result
-               (do
-                 (log/debug "No parameter changes could be retrieved. Querying for initial parameters")
-                 (let [initial-params-query {:select [[:initial-params.initial-param/key :param-change/key]
-                                                      [:initial-params.initial-param/db :param-change/db]
-                                                      [:initial-params.initial-param/value :param-change/value]
-                                                      [:initial-params.initial-param/set-on :param-change/applied-on]]
-                                             :from [:initial-params]
-                                             :where [:and [:= key :initial-params.initial-param/key]
-                                                     [:= db :initial-params.initial-param/db]]}]
-                   (paged-query initial-params-query
-                                first
-                                (when after
-                                  (js/parseInt after))))))]
-     ret)))
+                                             (js/parseInt after)))]
+     param-changes-result)))
```

Comment:
This resolver is for retrieving changes with all their history. Why not write a simple `current-parameters-query-resolver [:key :db]` resolver that just runs queries over`:param-changes` table for the client? Faster and with less room for an error.

Reply:
Sorry, not following here, the params value are coming from parameters table, this is for param changes where there is nothing as "current param changes"

Reply:
I though we talked about having two endpoints in the end - one for parameter changes one for current parameter values. I have looked at the client code and it actually looks like this endpoint is called just once, at boot and persisted in re-frame db, so it's all good I guess.